rag/README.md

```diff
@@ -0,0 +1,76 @@
+# Steps to create the RAG tool
+
+This readme file documents two steps:
+1) Deploy a standalone milvus container and initiate it with company documents
+2) Deploy a RAG API service that can be used as a tool by the Agent
+
+## 1. [Milvus](https://milvus.io/docs/) Deployment and [pymilvus](https://pymilvus.readthedocs.io/) SDK Usage
+
+This guide explains how to deploy Milvus using Kubernetes (specifically OpenShift), connect to it using the `pymilvus` SDK, and run operations on Milvus through a Jupyter Notebook.
+
+### Prerequisites
+
+1. **Kubernetes Cluster**: Ensure you have access to a Kubernetes cluster (e.g., OpenShift).
+2. **kubectl/oc CLI**: Command-line interface to manage Kubernetes/OpenShift resources.
+
+### Deployment Steps
+
+#### 1. Apply Milvus Deployment YAML with the following command:
+
+   ```bash
+   kubectl apply -f vector_db/standalone_milvus.yaml
+   ```
+
+#### 2. Connecting to Milvus with pymilvus
+
+#### Example Notebook Code to Delete a Collection
+
+1. Open Jupyter Notebook `vector_db/vector_db_insert.ipynb` and add the Milvus host from step 1.
```

Comment:
Could you also add a line mentioning what docs can be inserted into the vector db and pointing to the `/docs` folder that the users can use as examples?